apps/keyboard/js/keyboard/target_handlers_manager.js

```diff
@@ -105,7 +106,12 @@ TargetHandlersManager.prototype._callTargetAction = function(action,
     }
   }
 
-  handler[action](press);
+  this.promiseQueue = this.promiseQueue.then(function() {
+    handler[action](press);
+  }).catch(function(e) {
+    console.warn('TargetHandlersManager: ' +
+        'Error occurred for ' + action + ': ' + e);
```

Comment:
`': ', e);` So we could see the stack instead of casting the error object to string.